test/ClangImporter/objc_factory_method.swift

```diff
@@ -22,8 +22,7 @@ func testInstanceTypeFactoryMethodInherited() {
   _ = NSObjectFactorySub() // okay, prefers init method
   _ = NSObjectFactorySub(integer: 1)
   _ = NSObjectFactorySub(double: 314159)
-  _ = NSObjectFactorySub(float: 314159) // expected-error{{argument labels '(float:)' do not match any available overloads}} 
-  // expected-note @-1 {{overloads for 'NSObjectFactorySub' exist with these partially matching parameter lists: (integer: Int), (double: Double)}}
+  _ = NSObjectFactorySub(float: 314159) // expected-error{{incorrect argument label in call (have 'float:', expected 'integer:')}}
```

Comment:
Why does 'integer:' make more sense as a suggestion here than 'double:'?

Reply:
As far as I understand it's `integer:` because solution with Int uses default literal type in this case, where `double:` doesn't.